test/IntegrationTests/MongoDBTests.cs

```diff
@@ -28,16 +46,52 @@ public void SubmitsTraces(string packageVersion)
         const int spanCount = 3;
         for (int i = 0; i < spanCount; i++)
         {
-            collector.Expect("MongoDB.Driver.Core.Extensions.DiagnosticSources");
+            collector.Expect(MongoDBInstrumentationScopeName);
         }
 
+        collector.Expect(MongoDBInstrumentationScopeName, span => ValidateSpan(span));
```

Comment:
For older versions of MongoDb, is all of the data the same before and after this change? If not, are we concerned about any attributes changing?